test/unit/logic/community/utils.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { CommunityRole } from '../../../../src/types'
+import {
+  toCommunityWithUserInformationAndVoiceChat,
+  CommunityPrivacyEnum,
+  AggregatedCommunityWithMemberAndFriendsData,
+  CommunityVoiceChatStatus
+} from '../../../../src/logic/community'
+import { Profile } from 'dcl-catalyst-client/dist/client/specs/lambdas-client'
+import { createMockProfile } from '../../../mocks/profile'
+
+describe('Community Utils', () => {
+  describe('toCommunityWithUserInformationAndVoiceChat', () => {
+    const mockCommunity: AggregatedCommunityWithMemberAndFriendsData = {
+      id: 'test-community-id',
+      name: 'Test Community',
+      description: 'Test Description',
+      ownerAddress: '0xowner',
+      privacy: CommunityPrivacyEnum.Public,
+      active: true,
+      ownerName: 'Test Owner',
+      isHostingLiveEvent: false,
+      role: CommunityRole.Member,
+      membersCount: 10,
+      friends: ['0xfriend1', '0xfriend2']
+    }
+
+    const mockVoiceChatStatus: CommunityVoiceChatStatus = {
+      isActive: true,
+      participantCount: 5,
+      moderatorCount: 2
+    }
+
+    const profilesMap = new Map<string, Profile>([
+      ['0xfriend1', createMockProfile('0xfriend1')],
+      ['0xfriend2', createMockProfile('0xfriend2')]
+    ])
+
+    describe('when community is public', () => {
+      it('should include voiceChatStatus for members', () => {
+        const community = { ...mockCommunity, privacy: CommunityPrivacyEnum.Public, role: CommunityRole.Member }
+        const result = toCommunityWithUserInformationAndVoiceChat(community, profilesMap, mockVoiceChatStatus)
+
+        expect(result.voiceChatStatus).toEqual(mockVoiceChatStatus)
+      })
+
+      it('should include voiceChatStatus for non-members', () => {
+        const community = { ...mockCommunity, privacy: CommunityPrivacyEnum.Public, role: CommunityRole.None }
+        const result = toCommunityWithUserInformationAndVoiceChat(community, profilesMap, mockVoiceChatStatus)
+
+        expect(result.voiceChatStatus).toEqual(mockVoiceChatStatus)
+      })
+
+      it('should include voiceChatStatus for moderators', () => {
+        const community = { ...mockCommunity, privacy: CommunityPrivacyEnum.Public, role: CommunityRole.Moderator }
+        const result = toCommunityWithUserInformationAndVoiceChat(community, profilesMap, mockVoiceChatStatus)
+
+        expect(result.voiceChatStatus).toEqual(mockVoiceChatStatus)
+      })
+
+      it('should include voiceChatStatus for owners', () => {
+        const community = { ...mockCommunity, privacy: CommunityPrivacyEnum.Public, role: CommunityRole.Owner }
+        const result = toCommunityWithUserInformationAndVoiceChat(community, profilesMap, mockVoiceChatStatus)
+
+        expect(result.voiceChatStatus).toEqual(mockVoiceChatStatus)
+      })
+    })
```

Comment:
Could you move the descriptions of the different test contexts to the statements in the `describe` blocks?

For example:

```js
  describe('and the user is a member', () => {
      it('should include voiceChatStatus', () => {
        const community = { ...mockCommunity, privacy: CommunityPrivacyEnum.Public, role: CommunityRole.Member }
        const result = toCommunityWithUserInformationAndVoiceChat(community, profilesMap, mockVoiceChatStatus)

        expect(result.voiceChatStatus).toEqual(mockVoiceChatStatus)
      })
  })
```